src/Ticket.php

```diff
@@ -4836,10 +4836,20 @@ public static function showCentralList($start, $status = "process", bool $showgr
                         $options['criteria'][0]['value']      = Session::getLoginUserID();
                         $options['criteria'][0]['link']       = 'AND';
 
-                        $options['criteria'][1]['field']      = 12; // status
-                        $options['criteria'][1]['searchtype'] = 'equals';
-                        $options['criteria'][1]['value']      = 'process';
-                        $options['criteria'][1]['link']       = 'AND';
+                        $options['criteria'][5]['field']      = 12; // status
+                        $options['criteria'][5]['searchtype'] = 'equals';
+                        $options['criteria'][5]['value']      = 2;
+                        $options['criteria'][5]['link']       = 'AND';
+
+                        $options['criteria'][7]['link']       = 'OR';
+                        $options['criteria'][7]['criteria'][0]['link']       = 'AND';
+                        $options['criteria'][7]['criteria'][0]['field']      = 12; // status
+                        $options['criteria'][7]['criteria'][0]['searchtype'] = 'equals';
+                        $options['criteria'][7]['criteria'][0]['value']      = 1;
+                        $options['criteria'][7]['criteria'][1]['link']       = 'AND';
+                        $options['criteria'][7]['criteria'][1]['field']      = 5; // users_id_assign
+                        $options['criteria'][7]['criteria'][1]['searchtype'] = 'equals';
+                        $options['criteria'][7]['criteria'][1]['value']      = Session::getLoginUserID();
```

Comment:
Use the same filter as for the title:

![image](https://github.com/glpi-project/glpi/assets/8530352/2bcb1475-2781-417a-ac09-6580b72db582)